include/solo5/solo5.h

```diff
@@ -277,4 +277,14 @@ solo5_result_t solo5_block_write(solo5_off_t offset, const uint8_t *buf,
  */
 solo5_result_t solo5_block_read(solo5_off_t offset, uint8_t *buf, size_t size);
 
+/*
+ * Set the TLS base register. This sets the %fs segment register in
+ * x86_64 or the TPIDR_EL0 register in aarch64.
+ *
+ * This function fails in "spt" with SOLO5_R_EINVAL when "base"
+ * points to an invalid address. Virtio and hvt always return
+ * SOLO5_R_OK.
```

Comment:
No need to enumerate details of what the backends do in this comment. Either drop it or write something generic like "Solo5 implementations MAY return SOLO5_R_EINVAL if the (base) does not satisfy arch-specific requirements."

Reply:
Changed to the suggested comment.